docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+    "|  $S^{--}_{t,\\lambda}$   |  corrected sample data of -- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction    |Row 1   |\n",
+    "|  $p^{p|a}$   |  pressure of $^3$He-polarizer/analyzer  |  scalar, 1-dim   | from metadata  |Row 1   |\n",
+    "|  $l^{p|a}$   |  length of $^3$He-polarizer/analyzer   |  scalar, 1-dim   | from metadata   |Row 1   |\n",
+    "|  $T^{g, p|a}(\\lambda)$   |  transmission of empty glass of polarizer/analyzer   |  scalar, 1-dim   | from metadata  |Row 1   |\n",
+    "|  $O^{p|a}(\\lambda)=O^{0, p|a}*\\lambda$   |  wavelength-dependent opacity of $^3$He-polarizer/analyzer  |  continuous function of $\\lambda$   | $O^{0, p|a}$ fit to data |Row 1   |\n",
+    "|  $O^{0, p|a}$   |  wavelength-independent pre-factor of opacity of $^3$He-polarizer/analyzer   |  scalar, 1-dim   | fit to data  |Row 1   |\n",
+    "|  $P^{3He, p|a}(t) = C^{p|a}e^{-t/T^{1, p|a}}$   |  time-dependent nuclear polarization of $^3$He-polarizer/analyzer  |  continuous function of $t$   | $C^{p|a}, T^{1, p|a}$ fit to data   |Row 1   |\n",
+    "|  $C^{p|a}$   |  Pre-factor of nuclear polarization exponential decay of $^3$He-polarizer/analyzer  |  scalar, 1-dim   | fit to data | $P_{He0}$   |\n",
+    "|  $T^{1, p|a}$   |  Decay constant of nuclear polarization exponential decay of $^3$He-polarizer/analyzer  |  scalar, 1-dim   | fit to data |Row 1   |\n",
+    "|  $I^{DB,unpol-in,p|a}_{\\lambda, t}$   |  Intensity of direct-beam of an unpolarized-incoming beam to the polarized $^3$He-polarizer/analyzer  |  data array with columns for $I$, $\\lambda$, $t$   | neutron data  |Row 1   |\n",
+    "|  $I^{DB,depol,p|a}_{\\lambda}$   |  Intensity of direct-beam of an unpolarized-incoming beam to the depolarized $^3$He-polarizer/analyzer |  data array with columns for $I$, $\\lambda$  | neutron data  |Row 1   |\n",
+    "|  $I^{DB, no-cell}_{\\lambda}$   |  Intensity of direct-beam of an unpolarized-incoming beam to detector without passing cells or sample  |  data array with columns for $I$, $\\lambda$  | neutron data   |Row 1   |\n",
+    "|  $\\hat{T}^{\\pm, p|a}(t,\\lambda)$   |  Transmission-function of a in-coming polarized neutron beam into $^3$He-polarizer/analyzer  |  Row 1   | by inserting  $P^{3He, p|a}(t)$,  $O^{p|a}(\\lambda)$ |Row 1   |\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6301a57b-ae5f-41c2-8cf2-63e0e7533db3",
+   "metadata": {},
+   "source": [
+    "1) Overview and general workflow\n",
+    "2) Readout of test-data\n",
+    "3) Definition of ROI\n",
+    "4) Data correction"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "e7400275-ef73-47a7-a72e-d2cdbfa1378d",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "\n",
+    "### Overview and general workflow\n",
```

Comment:
This whole cell does not render for me (some math mode error about `$`). Does it work for you?

I can fix it by wrapping all the "equation" environments into $$....$$.

Reply:
strange, for me it works fine and looks nice as it is

Reply:
But to avoid display problems - what exactly you mean by wrapping into ....?

Reply:
```
$$
...
$$
```

Reply:
Ah sorry, in the code review it looked like just "..."